### ISSUE TITLE
Is there an .fld for R-1141?

### ISSUE THREAD

**Issue by adrianfernandez00** [usnistgov/REFPROP-wrappers#401]
That's it. I'm using REFPROP 10 and I would like to use R-1141 but I don't find it.

**Comment by ianhbell** [comment]
Nope.  There are three kinds of data in NIST TDE, certainly not enough to build an EOS.
